src/Jaeger/Configuration.cs

```diff
@@ -83,9 +83,9 @@ public class Configuration
         public const string JaegerSamplerParam = JaegerPrefix + "SAMPLER_PARAM";
 
         /// <summary>
-        /// The sampler manager host:port.
+        /// The url for the remote sampling conf when using sampler type remote.
         /// </summary>
-        public const string JaegerSamplerManagerHostPort = JaegerPrefix + "SAMPLER_MANAGER_HOST_PORT";
+        public const string JaegerSamplingEndpoint = JaegerPrefix + "SAMPLING_ENDPOINT";
```

Comment:
Usually we make these changes backward compatible by supporting the previous configuration for some period of time.

Reply:
I only looked at the CPP implementation and saw that it was renamed. I had not seen the discussion there that it was not implemented before. I will add it on Monday and mark it deprecated.

Reply:
Ah, the reason it was a straight renaming in CPP is because the env var there was not used at all, so the functionality was never released and never available to users of the library.

Reply:
I just readded the support and marked it as deprecated. If someone is using it, the logging will produce a warning at runtime.